src/main/business/applicationFacade.js

```diff
@@ -364,6 +387,31 @@ class ApplicationFacade{
             keywords: bAppendix.getKeywords(),
         }
     }
+
+    /* returns
+    {
+        answer,
+        stem,
+        appendix?
+    }
+     */
+    #generateTagAnswerTask(data) {
+        return {
+            answer: {
+                id: 1,
+                tag: ANSWER_TYPES.DISTRACTOR,
+                text: '0',
+                explanation: '',
+            },
+            stem: '$e^{i\\pi} + 1 = $',
+            appendix: {
+                title: "Euler's identity: ",
+                tag: "euler identity",
+                content: "\\setlength{\\fboxsep}{10pt} % Set the padding (default is 3pt)\n"
+                    + "\\fbox{\\huge $e^{i\\theta} = \\cos{\\theta} + i\\sin{\\theta}$}",
+            },
+        }
+    }
```

Comment:
Same

Reply:
this is example return for debugging purposes, also outdated